Packs/Whois/Integrations/Whois/Whois.yml

```diff
@@ -1,19 +1,25 @@
 category: Data Enrichment & Threat Intelligence
 commonfields:
-  id: Whois
+  id: Whois ip
```

Comment:
Why are we changing the id name and display?

Reply:
It was just for testing